### DISCUSSION TITLE
Welcome to twsnmpfk Discussions!

### DISCUSSION THREAD

**Discussion by twsnmp** [twsnmp/twsnmpfk#1] [Announcements]
## 👋 Welcome! ようこそ

The twsnmpfk Discussions is a place for everyone to think about the functionality of the TWSNMP.
Please let us know your questions and requests about TWSNMP.

twsnmpfk Discussionsは、TWSNMPの機能についてみんなで考える場です。
TWSNMPについての質問や要望をお知らせください。

**Comment by snowhite2501** [comment]
お世話になっております。
表示上の問題の報告です。

（動作環境）
v.1.23.0を使用
Windows 11 64bit

（発生している問題）
[設定]>[通知]の「通知間隔(秒)」は"分"で動作している？

細かく"秒"で指定できてもよいですが、少し気になりまして。
通知機能は"秒"と"分"ではどちらがうれしいのでしょうね？
私は"分"でも十分かなと思いますが。
（私の環境の問題でメール通知が大幅に遅れているだけでしたらすみません。）

TWSNMPにはお世話になってますのでGitHub自体にも初めて書き込んでみました。

**Reply by twsnmp** [reply]
ご連絡ありがとうございます。
![image](https://github.com/user-attachments/assets/baebf389-055a-41b6-ac31-48bb2953ebb8)

ですね。
分の間違いです。次回修正します。
